magefile.go

```diff
@@ -171,23 +149,3 @@ func Clean() error {
 
 	return os.RemoveAll("package-registry")
 }
-
-func Vendor() error {
-	fmt.Println(">> mod - updating vendor directory")
-
-	err := sh.RunV("go", "mod", "tidy")
-	if err != nil {
-		return err
-	}
```

Comment:
We probably still want to check that `go mod tidy` doesn't produce changes.

Reply:
Fixed, thanks